app/Services/Transactions/Aggregates/LargestTransactionAggregate.php

```diff
@@ -30,6 +30,7 @@ public function aggregate(): ?Transaction
         ->limit(1)
         ->first();
 
+        /* @phpstan-ignore-next-line */
         if ($transfer['amount'] >= $multipayment['multipayment_amount']) {
```

Comment:
Can you do the below instead of treating it like an array:

```suggestion
        if ($transfer->amount >= $multipayment->multipayment_amount) {
```